schemata/pdf_manager_template.json

```diff
@@ -3,7 +3,6 @@
   "id": "/schemata/pdf_manager_template",
 
   "title": "PDF Manager Template",
-  "stability": "prototype",
```

Comment:
We do nothing with this value right now, changing it means I have to update all others files because none of them use a stability other than prototype.